pageserver/src/tenant/storage_layer/delta_layer.rs

```diff
@@ -969,7 +969,11 @@ impl DeltaLayerInner {
                 .as_slice()
                 .iter()
                 .filter_map(|(_, blob_meta)| {
-                    if blob_meta.key.is_rel_dir_key() || blob_meta.key == DBDIR_KEY {
+                    if blob_meta.key.is_rel_dir_key()
+                        || blob_meta.key == DBDIR_KEY
+                        || blob_meta.key == AUX_FILES_KEY
```

Comment:
AUX_FILES_KEY is only used with aux files v1. It's the directory of what files exist. It should not be large.

Reply:
As far as I remember, the first problem we have faced with AUX was larger number of LR snapshot files.
So even AUX directory can be large.
It seems to be safer to skip AUX_FILES_KEY key here...